scripts/_package.json

```diff
@@ -15,6 +16,28 @@
   "bugs": {
     "url": "https://github.com/IgniteUI/igniteui-webcomponents/issues"
   },
+  "keywords": [
+    "webcomponents",
+    "web components",
+    "UI components",
+    "avatar",
+    "badge",
+    "bottom nav",
```

Comment:
We don't have a bottom nav yet.

Reply:
I thought the nav drawer serves this purpose with a bottom position, since the mini template is always visible.

Reply:
Actually, it's exactly the opposite - the mini is never shown when position is top or bottom.